From f043c84b9179ae4e50dc4d2ad7370c87cd353759 Mon Sep 17 00:00:00 2001
From: Demon000 <demonsingur@gmail.com>
Date: Thu, 11 Jan 2018 08:28:39 +0200
Subject: [PATCH 05/20] legacy: add back radio rules

In commit 625252b7aed77bcc69fb2c0fee619b9399a88dd8 some
radio rules have been removed that are still needed for
old blobs. Add them back.

Change-Id: Ia681415fcf641af9f70f5870edd1d9c60c295ba8
---
 legacy-common/radio.te | 7 +++++++
 1 file changed, 7 insertions(+)
 create mode 100644 legacy-common/radio.te

diff --git a/legacy-common/radio.te b/legacy-common/radio.te
new file mode 100644
index 0000000..01fafe6
--- /dev/null
+++ b/legacy-common/radio.te
@@ -0,0 +1,7 @@
+# Talks to qmuxd via the qmux_radio socket
+qmux_socket(radio)
+
+# Radio apps needs to communicate with ims sockets
+unix_socket_connect(radio, ims, ims)
+
+allow radio self:socket create_socket_perms_no_ioctl;
-- 
2.7.4

